generate/templates/partials/configurable_callbacks.cc

```diff
@@ -1,3 +1,4 @@
+#include <git2/oid.h>
```

Comment:
valid point. I didn't really like the solution of needing to make it specific to OIDs in the first place but I couldn't find anything better at the time. I'll give it another look to see if it can be made generic but if not I'll take this suggestion.

Reply:
I couldn't come up with anything better. maybe something to look into later.
the header file was already being included by the dependency inclusion template in struct_content.